src/prefect/task_engine.py

```diff
@@ -99,6 +107,12 @@
 BACKOFF_MAX = 10
 
 
+def get_labels_from_context(context: Optional[FlowRunContext]) -> Dict[str, Any]:
+    if context is None:
+        return {}
+    return context.flow_run.labels
```

Comment:
I don't think this is useful enough to be a separate function. Can you inline this in the two places where it's used?